MovieApp/MovieApp/App/LocalDataSource/MovieLocalDataSource.swift

```diff
@@ -30,7 +30,29 @@ class MovieLocalDataSource: MovieLocalDataSourceProtocol {
 
         try? realm.write {
             realm.delete(oldMovies)
-            realm.add(mappedMovies)
+            realm.add(mappedMovies, update: .all)
+        }
+    }
+
+    func saveLocal(movie: MovieDataModel, category: CategoriesDataSource) {
```

Comment:
No need for `local`

Reply:
Deleted this part, because I wasn't even using it, but renamed `deleteLocal` to `delete`